### ISSUE TITLE
Conflict stops refresh

### ISSUE THREAD

**Issue by Riveloper** [hjdhjd/homebridge-myq#323]
**Is your feature request related to a problem? Please describe:**

It appears there is a conflict with the SmartThings plugin. Possibly others with similar features where the refresh is stopped when the period refresh to SmartThings occurs and does not resume.

**Describe the solution you'd like:**
Modify how the refresh is monitored to detect if stopped.

**Describe alternatives you've considered:**
Show last update time as part of the accessory to make it more obvious that it has been stopped.

**Additional context:**

**Comment by hjdhjd** [comment]
Thanks for the report - you’re on your own I’m afraid, if you choose to use multiple unofficial plugins with the myQ ecosystem. I would highly advise against it for numerous reasons, not the least of which is the probability of having your account locked out due to overuse of the API.

My suggestion is that you use a single plugin to interact with the myQ API, and use a different mechanism to federate those updates, such as MQTT. This plugin does support MQTT, but I’m sure others do as well.

TL;DR: this is an unsupported use case, you’re on your own I’m afraid.

**Comment by Riveloper** [comment]
The Smarthings plugin does not access MyQ or Homebridge accessories.

It connects to Smartthings api to display those devices in Homebridge.

I just noticed that the MyQ logs ran for several hours then abruptly stopped, often after the Smarthings api attempted to refresh, interrupting the 13 second MyQ refresh from the plug-in.

Great work either way. And it is not a major problem. If others haven't had an issue, I just wanted to pass along my observation

**Comment by hjdhjd** [comment]
Thanks, appreciate the insight, but I’m a bit skeptical about the conclusions. Correlation isn’t causation...and I have no idea what SmartThings does (or doesn’t) do in the equation. Best of luck!
